scl/syslog-ng.conf

```diff
@@ -3,7 +3,7 @@
 # single file called /var/log/messages.
 #
 
-@version: 3.28
+@version: current 
```

Comment:
Please don't do this in our default configuration. It is our goal to have a proper @version in user configurations.

Reply:
I'll modify it.
IMO users can easily change it to a specific version...
What are the situations when a user want to change `@version` ?
When we do a change which is not compatible with the previous versions - in this case we show a warning message and in a later version we will break compatibility. In this case this warning is showed as expected - user can replace the `current`  with a specific version... so it requires the very same act from the user.